src/services/ApiClient.ts

```diff
@@ -35,6 +35,23 @@ interface GoldenRecordResponse {
   expandedGoldenRecords: GoldenRecord[]
 }
 
+interface JempiGoldenRecord extends Pick<PatientRecord, 'sourceId' | 'uid'> {
```

Comment:
@IlyesBenAmara rename MpiGoldenRecord to ExpandedGoldenRecord and JempiGoldenRecord to GoldenRecord